### PR TITLE
Fix multiple cli and backend bugs

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
Fix https://github.com/rancher/rio/issues/616 https://github.com/rancher/rio/issues/608 https://github.com/rancher/rio/issues/603 https://github.com/rancher/rio/issues/602 https://github.com/rancher/rio/issues/630

File: cli/pkg/tables/service.go [hunk 1]
```diff
@@ -16,20 +16,19 @@ import (
...
 		{"WEIGHT", "{{.Service | formatWeight}}"},
 		{"CREATED", "{{.Service.CreationTimestamp | ago}}"},
 		{"DETAIL", "{{serviceDetail .Service .Pod}}"},
 	}, cfg)
 
 	writer.AddFormatFunc("image", FormatImage)
 	writer.AddFormatFunc("scale", formatRevisionScale)
-	writer.AddFormatFunc("app", app)
+	writer.AddFormatFunc("appAndVersion", appAndVersion)
 	writer.AddFormatFunc("version", version)
```
Comment (cbron) [inline cli/pkg/tables/service.go]: Do we still need version ?

File: modules/rdns/controllers/service/handler.go [hunk 2]
```diff
@@ -183,7 +183,7 @@ func (h *handler) staticAddress() ([]adminv1.Address, error) {
 func (h *handler) getAddresses(svc *corev1.Service) ([]adminv1.Address, bool, error) {
 	result, err := h.staticAddress()
 	if len(result) > 0 || err != nil {
-		return result, false, err
+		return result, true, err
```
Comment (cbron) [inline modules/rdns/controllers/service/handler.go]: this is ok ?

Comment (StrongMonkey) [reply modules/rdns/controllers/service/handler.go]: This is the fix for RKE bug, when IP-address is set nodeport must be returned to true.

File: cli/pkg/tables/service.go (hunk 1 above)
Comment (StrongMonkey) [reply cli/pkg/tables/service.go]: whoops, don't need it anymore

**Pull request merged**